plugins/Updater/page1.c

```diff
@@ -24,7 +24,7 @@
 
 static TASKDIALOG_BUTTON TaskDialogButtonArray[] =
 {
-    { IDOK, L"Check for updates" }
```

Comment:
This was probably used more for space filler 😄 

There isn't much content on the dialog and Check on its own feels too short. you can get away with extra text for the button in this case.

Reply:
Sure, since that long wording was my initial reason for refactoring and I rewrote the instruction's text, too, can we shorten this one, please?
As an argument: why not un-comment the longer content's text below instead of the short one? 😄

Reply:
> why not un-comment the longer text below

I didn't think clicking the button needed the longer instruction text 😛